src/ducks/billsMatching/Linker/billsToOperation/helpers.js

```diff
@@ -93,12 +94,26 @@ const sortedOperations = (bill, operations) => {
   return sortBy(operations, buildSortFunction(bill))
 }
 
+const getBillRegexp = bill => {
+  let regexpStr = bill.matchingCriterias && bill.matchingCriterias.labelRegex
+
+  if (!regexpStr) {
+    const [brand] = getBrands(
+      brand => brand.name === bill.vendor || brand.konnectorSlug === bill.vendor
+    )
+    regexpStr = brand ? brand.regexp : `\\b${bill.vendor}\\b`
```

Comment:
Should we check here that bill.vendor is defined ? Otherwise the regex would match everytime right ?

Reply:
Yes good catch, I'm going to add this check